docs/architecture/adr-083-out-of-band-sync.md

```diff
@@ -0,0 +1,73 @@
+# ADR 083: Supporting out of band state sync
+
+## Changelog
+
+- 2022-10-31: Initial Draft (@cmwaters)
+
+## Status
+
+Accepted
+
+## Context
+
+Whenever Tendermint begins, it initiates a handshake with the application via `Info` to gauge version compatibility and receive the last height and app hash of the application. Tendermint then begins a replay protocol which aims to sync the heights of Tendermint's state and block store and the application's current height. 
+
+When initially designed, one invariant was that the applications current height should never exceed
+Tendermint's height. The protocol would error and the node would shut down if this were to happen. It seemed
+appropriate at the time that Tendermint which would provide blocks either through block sync or consensus
+would always be ahead. However, in light of state sync and other protocols for bootstrapping an application to 
+a particular height, this invariant is no longer necessary.
+
+## Decision
+
+Allow applications to start with a bootstrapped state alongside an empty Tendermint instance using
+a subsystem of the state sync protocol. 
```

Comment:
If I'm understanding correct, this is a protocol for Tendermint to sync to the height of the application. It doesn't actually require anything from the application other than for the application to report height higher than Tendermint's height?

Reply:
Correct (although RPC endpoints in the statesync config do need to be configured)